.github/contributing/docs_guide.md

```diff
@@ -0,0 +1,57 @@
+# Documentations Guide
+
+This guide aims to ensure consistency in the Bevy crates documentation, they should be seen as advice and not strict rules and should therefore be adapted on a case-by-case basis.
+
+## The crate documentation
+
+1. The first line is a single sentence that summarizes the crate role in the Bevy ecosystem.
+2. The first line is followed by an empty line.
+3. The documentation may contain more details.
+4. Links are placed at the end to clarify documentation reading.
+5. The crate attributes (`#![attribute]`) are placed _after_ the documentation.
```

Comment:
```suggestion
5. The crate attributes (`#![attribute]`) are placed _before_ the documentation. Consumers of the documentation are more likely to visit generated docs, rather than reading them in the source files.
```